bigframes/session/__init__.py

```diff
@@ -534,6 +534,11 @@ def _read_gbq_table_to_ibis_with_total_ordering(
         # the same assumption and use these columns as the total ordering keys.
         table = self.bqclient.get_table(table_ref)
 
+        if table.location.casefold() != self._location.casefold():
```

Comment:
Today I learned `casefold()`, thanks.

"Casefolded strings may be used for caseless matching." https://docs.python.org/3/library/stdtypes.html#str.casefold